saleor/graphql/notifications/inputs.py

```diff
@@ -0,0 +1,11 @@
+import graphene
```

Comment:
In other modules, we store inputs in `mutations.py` or `mutations\<file_name>.py`. IMO we should make it consistently. If we want to have inputs in a separate file we should do it for all modules. 
It could be a good idea if we want to reuse this input in multiple files. In this case, we use this input only in one mutations file.